tests/test_sorting.py

```diff
@@ -0,0 +1,367 @@
+import os
+import random
+import numpy as np
+from pathlib import Path
+import astropy.io.fits as fits
+
+from corgidrp import sorting as calsort
+import corgidrp.data as data
+from corgidrp.data import Image
+from corgidrp.mocks import create_default_headers
+
+
+# Sub-functions
+# NOTE: Most of the values for the different non-unity em gains and the number
+# of frames come from either TVAC or some preliminary version of the 
+# Commissioning test calculations
+
+def get_cmdgain_exptime_mean_frame(
+    exptime_sec=5,
+    nframes=30,
+    ):
+    """
+    Create an array of CMDGAIN, EXPTIME for frames that will be used to
+    generate a mean frame.
+
+    Args:
+      exptime_sec (float): exposure time of the frame in seconds
+      nframes (int): (minimum) number of frames to generate a mean frame
+    
+    Returns:
+      cmdgain_list (list): list of commanded gains
+      exptime_list (list): list of exposure frames
+    """
+    cmdgain_list = [1] * nframes
+    exptime_list = [exptime_sec] * nframes
+    return cmdgain_list, exptime_list
+
+def get_cmdgain_exptime_kgain(
+    exptime_sec=[0.077, 0.770, 1.538, 2.308, 3.077, 3.846, 4.615, 5.385, 6.154,
```

Comment:
this is minor, but python keywords shouldn't host non-primitive objects like lists. Instead the python best practice is to set the keyword to None e.g., `exptime_sec=None`, and then at the start of the function, have a block of code that is like `if exptime_sec is None: exptime_sec = [...]`.

Reply:
I set None and did not set the lists inside the functions because they must be defined for the assertions later on and it forces the user to know what's been set as a representative set of calibration data for K-gain, non-linearity and EM-gain vs DAC. 

The latter is important when adding more possible combinations in a post DRP-v1.1 review of the sorting algorithm of datasets than the one in the current test, which is based on TVAC and a draft version of the detailed implementation of the corresponding CAR (Commissioning Activity Report). It may change in the future and/or there are other distinct combinations to test